cmd/buildah/common.go

```diff
@@ -25,6 +25,11 @@ var (
 	needToShutdownStore = false
 )
 
+const (
+	maxPullPushRetries = 3
```

Comment:
These should probably go into containers.conf

Reply:
The retry semantics are specific to Buildah and are not implemented in other tools. I think that it should be implemented in containers/image and be configurable in the registries.conf.

Reply:
Ok let's merge and we can open a different PR if we decide we need this in registries.conf.